lang/en.json

```
@@ -674,9 +675,11 @@
     "SptAddGmPoint": "A GM Point has been added!",
     "SptSetGmPoints": "GM Points have been set to ",
     "SptSpendGmPoint": "A GM Point has been spent!",
+    "SptSpendGmPointDenied": "No GM Points to spend!",
     "SptAddStoryPoint": "A {name} has been added!",
     "SptSetStoryPoints": "{name} have been set to ",
     "SptSpendStoryPoint": "A {name} has been spent!",
+    "SptSpendStoryPointDenied": "No Story Points to spend!",
```

```suggestion
    "SptSpendStoryPointDenied": "No {name} Points to spend!",
```
You'll need to use `getPointsName()` to handle "Friendship Points", similar to title